addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -2,6 +2,10 @@
 <odoo>
     <data>
         <template id="message_notification_email">
+<html t-att-lang="lang"><header>
```

Comment:
isn't this removed by the HTMl sanitizer ?

Reply:
I think it is, so is this content not sanitized ?

![screenshot_spect](https://user-images.githubusercontent.com/54316226/167578276-6006bfc5-1976-45db-8546-81d227bbf14a.png)

Reply:
Well, I'm looking at the received emails (would you like a cc?). So, no, I suppose the rendered template is not sanitized in this way. Do you think it should, even if all the included contents should themselves be templates or have been sanitized?

Reply:
no I guess it's ok :) mail client are smart :)